addons/base_import/models/base_import.py

```diff
@@ -712,12 +712,18 @@ def _remove_currency_symbol(self, value):
             # Otherwise it is not a float with a currency symbol
             return False
 
+    def _convert_scientific_notation_to_float(self, value):
+        if not "e-" in value:
+            return value
+        return '{:f}'.format(float(value))
```

Comment:
Not sure this will work in all the cases, maybe it needs to be put after the replace of the separators ?
Test it with : 7.51×10−9 and with : 1,23 × 10−4

As you can see the separators are different.